purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -61,3 +72,37 @@ def _create_purchase_order(self, seller_id):
         po = self.env["purchase.order"].with_context(**ctx).create(vals)
         po._onchange_requisition_id()
         return po
+
+    def action_all_rfq_related_send(self):
+        self.ensure_one()
+        rfqs = self.purchase_ids.filtered(lambda x: x.state != "cancel")
+        template_id = self.env.ref("purchase.email_template_edi_purchase", False).id
+        ctx = dict(
+            self.env.context,
+            default_model="purchase.order",
+            active_model="purchase.order",
+            active_id=rfqs.ids[0],
+            active_ids=rfqs.ids,
+            default_template_id=template_id,
+            default_composition_mode="mass_mail",
+        )
+        return {
+            "type": "ir.actions.act_window",
+            "view_mode": "form",
+            "res_model": "mail.compose.message",
+            "target": "new",
+            "context": ctx,
+        }
```

Comment:
Now that the implementation is so much simplified (this is basically just an action), I'm giving it a 2nd thought and I think this is not the place where it should go. At least, not the *main* place.

We're just sending a batch email to a lot of RFQs, so the correct place IMHO would be as a context action item:

![imagen](https://user-images.githubusercontent.com/973709/232988622-2e5f28c4-8342-4aa9-93fe-426919af2701.png)

So, from the requisition, you just go to the list of RFQs, select the ones you want to mail, and mail them. The advantage would be that you could select only some RFQs and you could even trigger this action for RFQs that are not related to purchase requisitions.

This makes me think if this feature should be related to this module or if it should be in an independent module with just dependency on `purchase`. 🤔 

WDYT @fcvalgar @Shide @anddago78 @rafaelbn ?

Reply:
@yajo all suggested changes are resolved except one, I'm waiting @rafaelbn , for me your suggested change is the best solution...

Reply:
I'm thinking about the connection between modules, but this module should be able to be used alone (without https://github.com/OCA/sale-workflow/pull/2430 for example).

So I think the @yajo solution is the best approach.
If we need to add a button on the Purchase Requisition in the future, will be easily added if we develop this action first.

Reply:
> I'm thinking about the connection between modules, but this module should be able to be used alone (without https://github.com/OCA/sale-workflow/pull/2430 for example).

Correct, both modules are independent @anddago78 

So, what is missing here?